simulation/halsim_ws_client/src/main/native/cpp/HALSimWS.cpp

```diff
@@ -64,9 +64,12 @@ bool HALSimWS::Initialize() {
   if (uri != NULL) {
     m_uri = uri;
   } else {
-    m_uri = "/wpilibws";
+    m_uri = "/wpilibws/halsimws";
   }
 
+  constexpr auto baseSize = 10;
+  m_clientId = wpi::StringRef(m_uri.substr(baseSize));
```

Comment:
For the client version of the extension, we probably can leave the default `m_uri` unchanged, and just hardcode a clientId as a string constant (mainly because there will only ever be either 0 or 1 active connections)

Reply:
Want me to hardcode it to halsim_ws_client?

Reply:
Yeah that would be work